inc/classes/class-service-worker.php

```diff
@@ -95,4 +97,24 @@ public function cache_images( \WP_Service_Worker_Scripts $scripts ) {
 
 	}
 
+	/**
+	 * Enables offline google analytics.
+	 *
+	 * @param object $scripts
```

Comment:
`@param \WP_Service_Worker_Scripts $scripts Instance to register service worker behavior with.`

Reply:
@deepaklalwani97 We can merge after this change.